tailwind.config.ts

```diff
@@ -19,6 +19,7 @@ const config: Config = {
         background: "#FAFAFA",
         stroke: "#D4D4D8",
         black: "#27272A",
+        frost: "rgba(256,256,256,0.5)",
```

Comment:
This is white with 50% opacity, so we can just do `text-white/50` directly and remove this custom color ;) 🙌🏼